cmd/release-controller-api/http.go

```diff
@@ -1299,6 +1300,49 @@ func (c *Controller) apiReleaseConfig(w http.ResponseWriter, req *http.Request)
 	fmt.Fprintln(w)
 }
 
+func (c *Controller) apiAcceptedStreams(w http.ResponseWriter, req *http.Request) {
+	imageStreams, err := c.releaseLister.List(labels.Everything())
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	acceptedReleases := make(map[string][]string)
+
+	for _, stream := range imageStreams {
+		r, ok, err := releasecontroller.ReleaseDefinition(stream, c.parsedReleaseConfigCache, c.eventRecorder, *c.releaseLister)
+		if err != nil || !ok {
+			continue
+		}
+		if r.Config.EndOfLife {
+			continue
+		}
+		s := ReleaseStream{
+			Release: r,
+			Tags:    releasecontroller.SortedReleaseTags(r),
+		}
+		var tags []string
+		for _, tag := range s.Tags {
```

Comment:
Why not just feed `releasecontroller.SortedReleaseTags(r)` directly into the `range`?  No need to instantiate the `ReleaseStream`, but never really use it.

Reply:
Fixed